src/main/java/com/dynatrace/loadrunner/converter/FileScanner.java

```diff
@@ -0,0 +1,189 @@
+package com.dynatrace.loadrunner.converter;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+import com.dynatrace.loadrunner.Constants;
+
+public class FileScanner {
+
+	private final static char EOF = (char) -1;
+
+	private BufferedReader reader;
+
+	private char firstChar, secondChar, thirdChar;
+
+	private StringBuilder modifiedInstruction;
+	private StringBuilder unmodifiedInstruction;
+	private StringBuilder whiteSpace;
+	private StringBuilder newWhiteSpace;
+
+	private boolean outsideString = true;
+
+	public FileScanner(BufferedReader reader) {
+		this.reader = reader;
+	}
+
+	/**
+	 * 
+	 */
+	public void initialize() {
```

Comment:
why do we use here public method instead of initializing fields in constructor? It is also unclear to me why we use `getChar` in `initialize`. Could you please explain it? ;)